src/installer/corehost/cli/deps_entry.cpp

```diff
@@ -6,7 +6,45 @@
 #include "utils.h"
 #include "deps_entry.h"
 #include "trace.h"
+#include "bundle/runner.h"
 
+static pal::string_t normalize_dir_separator(const pal::string_t& path)
+{
+    // Entry relative path contains '/' separator, sanitize it to use
+    // platform separator. Perf: avoid extra copy if it matters.
+    pal::string_t normalized_path = path;
+    if (_X('/') != DIR_SEPARATOR)
+    {
+        replace_char(&normalized_path, _X('/'), DIR_SEPARATOR);
+    }
+
+    return normalized_path;
+}
+
+void deps_entry_t::append_resource_path(pal::string_t& base) const
+{
+    assert(asset_type == asset_types::resources);
+
+    pal::string_t pal_relative_path = asset.relative_path;
+    if (_X('/') != DIR_SEPARATOR)
+    {
+        replace_char(&pal_relative_path, _X('/'), DIR_SEPARATOR);
+    }
```

Comment:
`normalize_dir_separator()`?
```suggestion
    pal::string_t pal_relative_path = normalize_dir_separator(asset.relative_path);
```